presidio-anonymizer/tests/services/test_aes_cypher.py

```diff
@@ -14,6 +16,7 @@
         (b'1111111111111111', "面汤"),  # Chinese text
```

Comment:
If a string is a valid input which would get encoded as part of the process, should we have tests checking the validity of a string input as well?

Reply:
At this level only bytes are passed, added both string and bytes option in tests for upper level